tests/post_processing/test_iu_data.py

```diff
@@ -0,0 +1,146 @@
+import pandas as pd
+import pandas.testing as pdt
+import pytest
+from endgame_postprocessing.post_processing.iu_data import (
+    IUData,
+    InvalidIUDataFile,
+    preprocess_iu_meta_data,
+)
+
+from endgame_postprocessing.post_processing.disease import Disease
+
+
+def test_iu_data_get_priority_population_iu_missing_raises_exception():
+    with pytest.raises(Exception):
+        IUData(
+            pd.DataFrame({"IU_CODE": [], "Priority_Population_LF": []}),
+            disease=Disease.LF,
+        ).get_priority_population_for_IU("AAA00001")
+
+
+def test_iu_data_without_valid_priority_population_column_raises_exception():
+    with pytest.raises(Exception) as e:
+        IUData(
+            pd.DataFrame({"IU_CODE": [], "Priority_Population_InvalidDisease": []}),
+            disease=Disease.LF,
+        )
+    assert e.match(
+        "No priority population found for disease LF, expected Priority_Population_LF"
+    )
+
+
```

Comment:
💡 It might be good to have a test that verifies it will get the proper column name based on the disease, and/or throws an error for a disease that isn't implemented yet.